cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml

```diff
@@ -1,7 +1,9 @@
-{% set heapster_memory = "200Mi" -%}
+{% set metrics_memory = "200Mi" -%}
+{% set eventer_memory = "200Mi" -%}
 {% set num_nodes = pillar.get('num_nodes', -1) -%}
 {% if num_nodes >= 0 -%}
-  {% set heapster_memory = (200 + num_nodes * 3)|string + "Mi" -%}
+  {% set metrics_memory = (200 + num_nodes * 4)|string + "Mi" -%}
```

Comment:
Yes. Just in case. 
For 1000 node cluster we are able to operate without any sink at 2-2.3gb (with 30k pause and 2k system pods). GKE http output creating consumes +400MB.  That brings us to 2.75. If couple cases occur at once scraping, gke scraping and pod relist then the temporary memory consumption can be bigger. So to be on the VERY safe side I decided to go with 4 multpilier.

As we did a lot of noise (=events) during the tests the memory it became apparent that Eventer should have more memory. List that happens inside of watch consumes *LOTS* of memory (we will try to get rid of it just after 1.2).